### PR TITLE
update frontend import paths

### PR DESCRIPTION
**Pull request by hermannm**
### Changes

- change our custom import paths starting with `@` to `@/`, to avoid potential package conflicts
- add `~/` import path for top-level folders (e.g. `public`)

### CODE REVIEW

**Review by larwaa — CHANGES_REQUESTED** [review]
Otherwise LGTM

File: frontend/.eslintrc.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
     "import/order": [
       "error",
       {
-        "groups": ["builtin", "external", "internal", "parent", "sibling", "index", "object"],
+        "groups": ["builtin", "external", "internal", "unknown", "parent", "sibling", "index", "object"],
```
[the remark this answers is not in the document]
Comment (hermannm) [reply frontend/.eslintrc.json:41]: The alternative is to define `pathGroups` in our ESLint config, but that would entail having our path setup in two places.

[the remark this answers is not in the document]
Comment (larwaa) [reply frontend/.eslintrc.json]: Basically, when we go `eslint .` in the `frontend` directory, `frontend` is the root path, and so eslint resolves the imports defined in path based on that. However, your local eslint in vscode likely runs with a root path of just `.` or something along those lines, so the import resolution fails. Adding `project: ...` to eslintrc fixes it.

[the remark this answers is not in the document]
Comment (hermannm) [reply frontend/.eslintrc.json]: Great catch!

**Pull request merged**